meta-lmp-bsp/recipes-kernel/linux/linux-lmp-fslc-imx-rt_6.1.bb

```diff
@@ -1,10 +1,11 @@
-FILESEXTRAPATHS:prepend := "${THISDIR}/linux-lmp-fslc-imx:"
-
 include linux-lmp-fslc-imx_${PV}.bb
 
-KERNEL_REPO = "git://github.com/foundriesio/linux.git"
+# This should be after including non-RT recipe to override patches
+FILESEXTRAPATHS:prepend := "${THISDIR}/linux-lmp-fslc-imx-rt:"
```

Comment:
Here we can probably just use:

`FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}`

Reply:
@ricardosalveti fixed, thanks.